api/v1/models/endpoint.go

```diff
@@ -145,6 +150,25 @@ func (m *Endpoint) validateIdentity(formats strfmt.Registry) error {
 	return nil
 }
 
+func (m *Endpoint) validateLabels(formats strfmt.Registry) error {
+
+	if swag.IsZero(m.Labels) { // not required
+		return nil
+	}
+
+	if m.Labels != nil {
```

Comment:
why not
`if m.Labels == nil || swag.IsZero(m.Labels) {
    return nil
}`
It seems like it will be more readable and should be more efficient.